### DISCUSSION TITLE
Remix HTTP Handler after Node.js Fetch API

### DISCUSSION THREAD

**Discussion by peibolsang** [remix-run/remix#3395] [Q&A]
I was wondering if the Remix team has any plans to re-architect (even deprecate?) its [HTTP Handler](https://remix.run/docs/en/v1/pages/technical-explanation#http-handler-and-adapters) after Node.js announced the introduction of the `fetch` API based on the Web standard.

With this move, it seems all major Javascript engines (Node.JS, Deno, V8) and/or the platforms that use them (Vercel, CloudFlare, Deno Deploy, Netlify) will be based on the Web Fetch API, correct?

Also, [this type of interoperability seems where the world is going](https://techcrunch.com/2022/05/09/cloudflare-heads-effort-to-make-javascript-environments-interoperable/?guccounter=1)! 

Wouldn't these evolutions make Remix' HTTP Handler obsolete?

**Comment by sergiodxa** [comment]
Remix still supports older versions of Node, once they can support only versions with fetch support they may be able to remove their Fetch polyfill.

Also, note that the http handler you linked is used to use Remix as a request handler on Express, that will still be needed.
